scanner/src/test/assets/mime-types-2.1.18_scancode-2.9.7.json

```diff
@@ -0,0 +1,506 @@
+{
+  "scancode_notice": "Generated with ScanCode and provided on an \"AS IS\" BASIS, WITHOUT WARRANTIES\nOR CONDITIONS OF ANY KIND, either express or implied. No content created from\nScanCode should be considered or used as legal advice. Consult an Attorney\nfor any legal advice.\nScanCode is a free software code scanning tool from nexB Inc. and others.\nVisit https://github.com/nexB/scancode-toolkit/ for support and download.",
+  "scancode_version": "2.9.7",
+  "scancode_options": {
+    "input": "/home/sse1be/Development/mime-types",
```

Comment:
You might want to replace your home folder with a generic path.

Reply:
Thanks for pointing that out!